x-pack/plugins/ml/public/application/components/import_export_jobs/import_jobs_flyout/cannot_read_file_callout.tsx

```diff
@@ -21,10 +21,12 @@ export const CannotReadFileCallout: FC = () => {
         })}
         color="warning"
       >
-        <FormattedMessage
-          id="xpack.ml.importExport.importFlyout.cannotReadFileCallout.body"
-          defaultMessage="Please select a file contained Machine Learning jobs which have been exported from Kibana using the Export Jobs option"
-        />
+        <div data-test-subj="mlJobMgmtImportJobsFileReadErrorCallout">
+          <FormattedMessage
+            id="xpack.ml.importExport.importFlyout.cannotReadFileCallout.body"
+            defaultMessage="Please select a file contained Machine Learning jobs which have been exported from Kibana using the Export Jobs option"
```

Comment:
Not related to this PR, but noticed a typo in this message - should be `containing` not `contained`.